ci-operator/config/openshift/cluster-storage-operator/openshift-cluster-storage-operator-main.yaml

```diff
@@ -225,6 +225,14 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-aws-csi
+- as: hypershift-e2e-aks
+  steps:
+    cluster_profile: hypershift
+    env:
+      AUTH_THROUGH_CERTS: "true"
+      ENABLE_HYPERSHIFT_CERT_ROTATION_SCALE: "true"
+      TECH_PREVIEW_NO_UPGRADE: "true"
+    workflow: hypershift-azure-aks-e2e
```

Comment:
Are you sure you want this test to block merges? Looking at the number of rehearsals, it does not give me confidence this job is going to be stable.

Reply:
Yeah we don't want something to merge that would break ARO HCP. The number of rehearsals is likely due to the issues we ran into November & December with quotas on Azure; that's been fixed.

Recently this past week, CNO pushed a change that broke HyperShift on AWS & AKS, and there was the RHCOS/CRIO issue that was breaking everyone.

Reply:
Here's what our current job status looks like on our HyperShift repo on presubmits - https://prow.ci.openshift.org/job-history/gs/test-platform-results/pr-logs/directory/pull-ci-openshift-hypershift-main-e2e-aks?buildId=1879900550519066624

Reply:
Thanks, that looks much better.
/approve